static/app/types/alerts.tsx

```diff
@@ -291,3 +291,8 @@ export enum MonitorType {
   CONTINUOUS = 0,
   ACTIVATED = 1,
 }
+
+export enum ActivationConditionType {
+  RELEASE_CREATION = 0,
+  DEPLOY_CREATION = 1,
```

Comment:
Do the ints map to the database storage level? if so 👍 if not, maybe we could use strings to define these so `0` won't evaluate as falsy.

Reply:
[yessir they do indeed](https://github.com/getsentry/sentry/blob/master/src/sentry/incidents/utils/types.py#L13-L15)